lib/widgets/compose_box.dart

```diff
@@ -286,30 +286,47 @@ class _ContentInput extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    ColorScheme colorScheme = Theme.of(context).colorScheme;
-
-    return InputDecorator(
-      decoration: const InputDecoration(),
-      child: ConstrainedBox(
-        constraints: const BoxConstraints(
-          // TODO constrain this adaptively (i.e. not hard-coded 200)
-          maxHeight: 200,
-        ),
-        child: ComposeAutocomplete(
-          narrow: narrow,
-          controller: controller,
-          focusNode: focusNode,
-          fieldViewBuilder: (context) {
-            return TextField(
+    const verticalPadding = 8.0;
+    const contentLineHeight = 22.0;
+
+    final designVariables = DesignVariables.of(context);
+
+    return ConstrainedBox(
+      constraints: const BoxConstraints(
+        // The minimum height fits a little more than 2 lines to match the spec
```

Comment:
It would be pretty hard to adapt to multiple font sizes. We could use `minLines` on `TextField`, but not `maxLines`. Maybe we should bend the design and make the content input a bit taller.